### ISSUE TITLE
Load Model error? 

### ISSUE THREAD

**Issue by math-yyj** [zamling/PSALM#1]
I found the following warning/error when trying the demo. 

but not sure why Load Model with error.

Can soneone give some suggestions? 

Many thanks !

DETAIL LOG: 

loading segmentation model
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
current model is psalm
Mask Decoder has been trained, init directly
current seg concat mode: False, seg_norm: False, seg_proj: True, seg_fuse_score: False
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:01<00:00,  1.30it/s]Some weights of PSALM were not initialized from the model checkpoint at models and are newly initialized: ['model.layers.0.self_attn.k_proj.bias', 'model.layers.0.self_attn.k_proj.weight',layers.9.self_attn.q_proj.weight', 'model.layers.9.self_attn.v_proj.bias', 'model.layers.9.self_attn.v_proj.weight']
You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.

**Comment by math-yyj** [comment]
besides, the result is much lower than which reported in the paper, feel confused...

**Comment by zamling** [comment]
Hi, @math-yyj 
Some parameters are not loaded successfully, so that you get the wrong results
May be you need to check some package versions which may affect model loading
transformers==4.36.2 tokenizers==0.15.2 accelerate==0.21.0 safetensors==0.4.2

**Comment by zamling** [comment]
Hi, @math-yyj 
I reproduced the whole evaluation progress again and got the same results in paper

[2024-04-09 10:51:34,937] [INFO] [real_accelerator.py:110:get_accelerator] Setting ds_accelerator to cuda (auto detect)
loading segmentation model
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
Special tokens have been added in the vocabulary, make sure the associated word embeddings are fine-tuned or trained.
current model is psalm
Mask Decoder has been trained, init directly
current seg concat mode: False, seg_norm: False, seg_proj: True, seg_fuse_score: False
Loading checkpoint shards: 100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:01<00:00,  1.13it/s]
SemSegEvaluator(ignore_label) is deprecated! It should be obtained from metadata.
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 5000/5000 [1:03:12<00:00,  1.32it/s]
OrderedDict([('panoptic_seg', {'PQ': 55.80602931528503, 'SQ': 83.19438181344853, 'RQ': 66.29610025739716, 'PQ_th': 61.84441326889771, 'SQ_th': 84.25620571394121, 'RQ_th': 72.91460321833402, 'PQ_st': 46.691487498511194, 'SQ_st': 81.59162875610097, 'RQ_st': 56.305907108813145})])
OrderedDict([('sem_seg', {'mIoU': 66.61229878000674, ...

**Comment by zamling** [comment]
This issue has been solved by making transformers==4.36.2 and safetensors==0.4.2, and I'll close this issue
